src/models/Account.ts

```diff
@@ -3,7 +3,7 @@ import mongoose, { Schema } from 'mongoose';
 import { NetworkProvider } from '../util/network';
 import { encryptString } from '../util/encrypt';
 
-export type AccountDocument = mongoose.Document & {
+export interface IAccount {
```

Comment:
@JustinaMary Extracted this interface from the existing type so we can use for the param in the AccountService.update() method

Reply:
Adjusted this a bit more in a later commit that dedicated an interface for AccountUpdates specifically.